### DISCUSSION TITLE
Como vocês lidam com cards responsivos?

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#928] [Perguntas e Respostas]
Acredito que seja uma questão de UX, já peguei vários layouts que possuem uma lista de cards, por exemplo, cards são mostrados de 9 em 9, com uma paginação no final.

O que me incomoda é que no mobile, normalmente os cards são mostrados do mesmo tamanho que o desktop só que de 1 em 1, e como um card tem um tamanho relativamente "grande" ele acaba ocupando quase metade da tela, então o usuário começa a rolar o scroll eternamente até chegar o final.

 Na minha cabeça uma boa solução é diminuí-lo e inserir um botão para expandir o card. Qual a opinião de vocês sobre a usabilidade de cards no mobile?

**Comment by kvnol** [comment]
Olá, @heliomsolivas. Cara, também creio que diminuí-lo e expandir com o :hover (:active, no mobile) seja a melhor opção mesmo. Como no exemplo que montei: https://codepen.io/kvnol/pen/jZMwXq

O que mudaria é, claro, a altura que não seria definida e creio que seria necessário um overflow para não explodir o conteúdo. Mas vejo que daria para ser feito apenas com CSS.

**Comment by luizbills** [comment]
Não sou da área de UX, mas já vi muitos recomendando nunca ocultar nada para o usuário que seja interessante ou importante. E recomendo [este material](https://uxdesign.cc/design-better-cards-c0d12ab581c4) que me ensinou muita coisa útil sobre cards.

**Comment by FilhoSergio** [comment]
Utilizo o overflow também para não vazar o conteúdo .

**Comment by rafaelcavalcante** [comment]
@luizbills artigo do caralho! Agradeço! :)
